vue-config/config.default.js

```diff
@@ -13,10 +13,6 @@ module.exports = {
     }
   },
   configureWebpack: {
-    output: {
```

Comment:
Relative to your comment, did you remove this code because it was useless ? I assume the default chunk name of a file is its file name. Right ?

Reply:
It was not useless, it was setting bundle files names as following :

`home.js`

But the correct way to name resources files is : 

`home.8722982387.js`

Reply:
Yep, understood ! I missread your previous comment !